examples/model_hosting/notebook/simple_scheduled_calculation/SimpleScheduledCalculation.ipynb

```diff
@@ -0,0 +1,527 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "df = pd.DataFrame(prediction[\"timeSeries\"][\"mean\"], columns=[\"timestamp\", \"mean\"])\n",
+    "df.plot(x=\"timestamp\")"
```

Comment:
you can use the helper method ScheduleOutput(prediction).get_dataframe("mean") here